service/storage_proxy.hh

```diff
@@ -219,8 +220,11 @@ public:
     query::tombstone_limit get_tombstone_limit() const;
     inet_address_vector_replica_set get_live_endpoints(const locator::effective_replication_map& erm, const dht::token& token) const;
 
+    future<std::vector<dht::token_range_endpoints>> describe_ring(const sstring& keyspace, bool include_only_local_dc = false) const;
+
 private:
     distributed<replica::database>& _db;
+    gms::gossiper& _gossiper;
```

Comment:
It was deliberately removed not so long ago, don't put it back, please. There's gossiper reference on proxy::_remote, use it instead.

Reply:
oh, I didn't know about the remote, fixed